utilities/unittests/test_pytest_utils.py

```
@@ -1264,12 +1264,46 @@ def test_get_cnv_version_explorer_url_missing_env(self):
     def test_get_cnv_version_explorer_url_no_relevant_flags(self):
         """Test no action when no relevant flags are set"""
         mock_config = MagicMock()
-        mock_config.getoption.side_effect = lambda option: {"install": False, "upgrade": "regular"}.get(option, False)
+        mock_config.getoption.side_effect = lambda option: {
+            "install": False,
+            "upgrade": None,
+            "upgrade_custom": None,
+        }.get(option)
```

Ruff (0.15.21)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)